tests/puppeteer/campaigns/functional/BO/13_shopParameters/03_productSettings/02_productPage/06_displayDiscountedPrice.js

```diff
@@ -31,7 +31,7 @@ const priceRuleData = new PriceRuleFaker({
   reduction: 20,
 });
 const unitDiscountToCheck = '€20.00';
-const unitPriceToCheck = '€2.94';
+const unitPriceToCheck = '€8.68';
```

Comment:
Can you please add some comments so we understand better what prices are that ?

Reply:
Okey :)